index.html

```diff
@@ -181,6 +181,27 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script>
+  var fireballSize = 22;
+  var getFireballSpeed = function(left) {
```

Comment:
в данном случае тебе не нужно писать else
просто
```suggestion
var getFireballSpeed = function(left) {
    if (left) {
      return 5;
    } 
    return 2;
  };
```

Reply:
если отработает первое правило - return 5;
и функция завершится - иначе выполнится return 2;